src/smirnoffio.py

```diff
@@ -305,6 +321,36 @@ def readsrc(self, **kwargs):
         # Store a separate copy of the molecule for reference restraint positions.
         self.ref_mol = deepcopy(self.mol)
 
+    @staticmethod
```

Comment:
I'm fine with having this in FB but shouldn't this be an OpenFF feature? Is this here as a temp-fix because the feature isn't due to be released in OpenFF until some point in the future?

Reply:
> Is this here as a temp-fix because the feature isn't due to be released in OpenFF until some point in the future?

Unfortunately this is currently the case. @j-wags might have more information on a timeline for this however?